h/static/styles/annotations.scss

```diff
@@ -162,3 +178,70 @@ privacy {
     color: $text-color;
   }
 }
+
+.less-quote, .less-body {
+  color: rgba(189, 28, 43, 0.5);
+  font-style: italic;
+  font-family: Georgia,"Bitstream Charter","Times","Times New Roman",serif;
+
+  &:hover {
+    color: #BD1C2B;
+  }
+}
+
+.more-quote, .more-body {
+  color: rgba(189, 28, 43, 0.5);
+  background: #fff;
+  font-style: italic;
+  font-family: Georgia,"Bitstream Charter","Times","Times New Roman",serif;
+
+  &:hover {
+    color: #BD1C2B;
+  }
+}
+
+.gradient {
```

Comment:
Why a separate gradient class? It doesn't suggest to me, as a reader, what kind of gradient it is (is it horizontal? vertical? what color? etc etc) so I don't find it super usable.

If there isn't a time when we wish to have the excerpt more/less without a gradient then just use the gradient always, as I did in the original patch I posted.